Cargo.toml

```diff
@@ -62,3 +65,9 @@ nogc_lock_free = ["nogc", "lockfreeimmortalspace"]
 nogc_no_zeroing = ["nogc_lock_free"]
 
 single_worker = []
+
+# malloc implementations
```

Comment:
Probably explain only one of the malloc features can be selected. Otherwise we use default malloc from libc. And be clear that this does not replace the global allocator for Rust, but only provides those libraries for GC implementation.